src/Symfony/Component/EventDispatcher/Attribute/AsEventListener.php

```diff
@@ -20,9 +20,24 @@
 class AsEventListener
 {
     public function __construct(
+        /**
+         * The event name to listen to.
+         */
         public ?string $event = null,
-        public ?string $method = null,
-        public int $priority = 0,
+
+        /*
+         * The method that acts as the listener.
+         */
+        string $method = null,
+
+        /*
+         * The priority of this listener if several are declared for the same transition.
```

Comment:
We can explain how priority works and how to debug.
```suggestion
         * The priority of this listener if several are declared for the same transition.
         * Listeners with the highest priority are called first.
         * Use the "debug:event-dispatcher" command to show priorities of other listeners.
```